src/site/content/en/blog/optimize-lcp/index.md

```diff
@@ -1,668 +1,458 @@
 
-{% Aside %}
-For more details, refer to the [Minify CSS](/minify-css/) guide.
-{% endAside %}
+The goal in this step is to ensure the LCP resource starts loading as early as possible. While in theory the earliest a resource _could_ start loading is immediately after TTFB, in practice there is always some delay before browsers actually start loading resources.
+
+A good rule of thumb is that your LCP resource should start loading at the same time as the first resource loaded by that page. Or, to put that another way, if the LCP resource starts loading later than the first resource, then there's opportunity for improvement.
+
+{% Img src="image/eqprBhZUGfb8WYnumQ9ljAxRrA72/8mqXeiEsLQwjgq2lEbjn.png", alt="A network waterfall diagram showing the LCP resource starting after the first resource, showing the opportunity for improvement", width="800", height="375" %}
+
+Generally speaking, there are two factors that affect how quickly an LCP resource can be loading:
+
+- When the resource is discovered.
+- What priority the resource is given.
+
+#### Optimize when the resource is discovered
 
-### Defer non-critical CSS
+To ensure your LCP resource starts loading as early as possible, it's critical that the resource is discoverable by the browser from the initial HTML document response. For example, in the following cases, the browser can discover the LCP resource by scanning the HTML document response:
 
-Use the
-[Coverage](https://developer.chrome.com/docs/devtools/coverage/) tab
-in Chrome DevTools to find any unused CSS on your web page.
+- The LCP element is an `<img>` element, and its `src` or `srcset` attributes are present in the initial HTML markup.
+- The LCP element requires a CSS background image, but that image is preloaded via `<link rel="preload">` in the HTML markup (or via a `Link` header).
+- The LCP element is a text node that requires a web font to render, and the font is loaded via `<link rel="preload">` in the HTML markup (or via a `Link` header).
 
-{% Img
-  src="image/admin/wjS4NrU5EsJeCuvK0zhn.png",
-  alt="Coverage tab in Chrome DevTools",
-  width="800",
-  height="559"
-%}
+Here are some examples where the LCP resource cannot be discovered from scanning the HTML document response:
 
-To optimize:
+- The LCP element is an `<img>` that is dynamically added to the page via JavaScript.
+- The LCP element requires a CSS background image.
 
-- Remove any unused CSS entirely or move it to another stylesheet if used on a
-  separate page of your site.
-- For any CSS not needed for initial rendering, use
-  [loadCSS](https://github.com/filamentgroup/loadCSS/blob/master/README.md) to
-  load files asynchronously, which leverages `rel="preload"`and `onload`.
+In both of the last two cases, the browser needs to run the script or apply the stylesheet—which  usually involves waiting for network requests to finish—before it can discover the LCP resource and could start loading it. This is never optimal.
```

Comment:
```suggestion
In both of the last two cases, the browser needs to run the script or apply the stylesheet—which usually involves waiting for network requests to finish—before it can discover the LCP resource and could start loading it. This is never optimal.
```